contracts/governance/IGovernor.sol

```diff
@@ -216,6 +216,14 @@ abstract contract IGovernor is IERC165 {
         bytes32 descriptionHash
     ) public payable virtual returns (uint256 proposalId);
 
+    /**
+     * @dev Cancel a proposal. This is restricted to Pending proposal (before the vote starts) and is restricted to
+     * the proposal's proposer.
```

Comment:
This isn't really accurate here. The specific logic depends on the modules, for example it's different between Governor and GovernorCompatibilityBravo, so it should be documented in `Governor`.